.typos.toml

```diff
@@ -1,6 +1,7 @@
 [default.extend-words]
 mosquitto = "mosquitto"
 hda = "hda"
+tegde = "tedge"
```

Comment:
I guess you mean:

```suggestion
tedge = "tedge"
```

Reply:
No actually the key is the word to detect, and the value is the intended value.

For example with `tegde = "tedge"` will detect the incorrect spelling, otherwise it does not detect this spelling:

```
% ~/.cargo/bin/typos
error: `tegde` should be `tedge`
  --> ./docs/src/references/c8y-configuration-management.md:5:1
  |
5 | tegde
  | ^^^^^
  |
```

I believe it has to do with how `typos` is doing word splitting...it somehow thinks that both `tedge` and `tegde` are correct.

Reply:
That is how to original typo made it into the docs in the first place (because `typos` did not detect it)